Tools/autotest/models/Callisto.param

```diff
@@ -40,15 +55,51 @@ MOT_SPIN_MIN,0.2
 MOT_THST_EXPO,0.5
 MOT_THST_HOVER,0.52
 MOT_YAW_HEADROOM,50
+PILOT_ACCEL_Z,250
+PILOT_SPEED_DN,0
+PILOT_SPEED_UP,250
+PILOT_THR_BHV,3
+PILOT_THR_FILT,0
+PILOT_TKOFF_ALT,0
+PILOT_Y_EXPO,0
+PILOT_Y_RATE,60
+PSC_ACCZ_D,0
+PSC_ACCZ_FF,0
 PSC_ACCZ_FLTD,10
 PSC_ACCZ_FLTE,0
 PSC_ACCZ_FLTT,10
-PSC_ACCZ_I,0.5
-PSC_ACCZ_P,0.25
+PSC_ACCZ_I,1.0
+PSC_ACCZ_IMAX,800
+PSC_ACCZ_P,0.5
+PSC_ACCZ_SMAX,0
 PSC_ANGLE_MAX,45
+PSC_JERK_XY,5
+PSC_JERK_Z,10
 PSC_POSXY_P,1
 PSC_POSZ_P,0.5
+PSC_VELXY_D,0.5
+PSC_VELXY_FF,0
+PSC_VELXY_FLTD,5
+PSC_VELXY_FLTE,5
+PSC_VELXY_I,1
+PSC_VELXY_IMAX,1000
+PSC_VELXY_P,2
+PSC_VELZ_D,0
+PSC_VELZ_FF,0
+PSC_VELZ_FLTD,5
+PSC_VELZ_FLTE,5
+PSC_VELZ_I,0
+PSC_VELZ_IMAX,1000
 PSC_VELZ_P,2.5
+WP_NAVALT_MIN,0
+WP_YAW_BEHAVIOR,1
 WPNAV_ACCEL,250
+WPNAV_ACCEL_Z,100
+WPNAV_JERK,1
+WPNAV_RADIUS,200
+WPNAV_RFND_USE,0
 WPNAV_SPEED,1500
+WPNAV_SPEED_DN,150

```

Comment:
It looks like at least some of these parameters are the same as the default values.  We should try and avoid adding the defaults to parameter files where possible.